docs/source/getting_started/index.ipynb

```diff
@@ -1164,9 +1164,9 @@
    "id": "762799a5-5b99-46ca-8b68-401dfdf10100",
    "metadata": {},
    "source": [
-    "To request multiple years of data, you can include the years in the \"year\" parameter in the form of `[Start Year, Stop Year]`.\n",
+    "To request multiple years of data, you can include the years in the \"year\" parameter in the form of `[Start Year, Stop Year]`. Date ranges can only be used for multi-year datasets.\n",
     "\n",
-    "Date ranges can only be used for multi-year datasets"
+    "In `src.datasets` for a multi-year dataset, the column value for \"Year\" is \"MULTIPLE\", and the columns \"coverage_start\" to \"coverage_end\" specifies the dates that exist in a specific TableType data that spans multiple years."
```

Comment:
Added the second sentence per your suggestion. I think this is a separate point from the multi-year parameter sentence, so I merged 1169 with 1167 and added a new line to emphasize that the columns are for src.datasets